src/util/numeric.h

```diff
@@ -232,6 +232,8 @@ u64 murmur_hash_64_ua(const void *key, int len, unsigned int seed);
 bool isBlockInSight(v3s16 blockpos_b, v3f camera_pos, v3f camera_dir,
 		f32 camera_fov, f32 range, f32 *distance_ptr=NULL);
 
+s16 adjustDist(s16 dist, float fov);
```

Comment:
Argument doesn't match function in .cpp.
Should be `zoom_fov`.

Reply:
yep